tirocks-sys/bindings/bindings.rs

```diff
@@ -4831,3 +5057,329 @@ extern "C" {
+    #[doc = " There are 4 reasons currently."]
+    FLUSH_WRITE_BYTES = 83,
+    #[doc = " COMPACTION_KEY_DROP_* count the reasons for key drop during compaction"]
+    #[doc = " There are 4 reasons currently."]
+    NUMBER_DIRECT_LOAD_TABLE_PROPERTIES = 84,
+    #[doc = " COMPACTION_KEY_DROP_* count the reasons for key drop during compaction"]
+    #[doc = " There are 4 reasons currently."]
+    NUMBER_SUPERVERSION_ACQUIRES = 85,
+    #[doc = " COMPACTION_KEY_DROP_* count the reasons for key drop during compaction"]
+    #[doc = " There are 4 reasons currently."]
+    NUMBER_SUPERVERSION_RELEASES = 86,
+    #[doc = " COMPACTION_KEY_DROP_* count the reasons for key drop during compaction"]
+    #[doc = " There are 4 reasons currently."]
+    NUMBER_SUPERVERSION_CLEANUPS = 87,
+    NUMBER_BLOCK_COMPRESSED = 88,
+    NUMBER_BLOCK_DECOMPRESSED = 89,
+    NUMBER_BLOCK_NOT_COMPRESSED = 90,
+    MERGE_OPERATION_TOTAL_TIME = 91,
+    FILTER_OPERATION_TOTAL_TIME = 92,
+    ROW_CACHE_HIT = 93,
+    ROW_CACHE_MISS = 94,
+    READ_AMP_ESTIMATE_USEFUL_BYTES = 95,
+    READ_AMP_TOTAL_READ_BYTES = 96,
+    NUMBER_RATE_LIMITER_DRAINS = 97,
+    NUMBER_ITER_SKIP = 98,
+    BLOB_DB_NUM_PUT = 99,
+    BLOB_DB_NUM_WRITE = 100,
+    BLOB_DB_NUM_GET = 101,
+    BLOB_DB_NUM_MULTIGET = 102,
+    BLOB_DB_NUM_SEEK = 103,
+    BLOB_DB_NUM_NEXT = 104,
+    BLOB_DB_NUM_PREV = 105,
+    BLOB_DB_NUM_KEYS_WRITTEN = 106,
+    BLOB_DB_NUM_KEYS_READ = 107,
+    BLOB_DB_BYTES_WRITTEN = 108,
+    BLOB_DB_BYTES_READ = 109,
+    BLOB_DB_WRITE_INLINED = 110,
+    BLOB_DB_WRITE_INLINED_TTL = 111,
+    BLOB_DB_WRITE_BLOB = 112,
+    BLOB_DB_WRITE_BLOB_TTL = 113,
+    BLOB_DB_BLOB_FILE_BYTES_WRITTEN = 114,
+    BLOB_DB_BLOB_FILE_BYTES_READ = 115,
+    BLOB_DB_BLOB_FILE_SYNCED = 116,
+    BLOB_DB_BLOB_INDEX_EXPIRED_COUNT = 117,
+    BLOB_DB_BLOB_INDEX_EXPIRED_SIZE = 118,
+    BLOB_DB_BLOB_INDEX_EVICTED_COUNT = 119,
+    BLOB_DB_BLOB_INDEX_EVICTED_SIZE = 120,
+    BLOB_DB_GC_NUM_FILES = 121,
+    BLOB_DB_GC_NUM_NEW_FILES = 122,
+    BLOB_DB_GC_FAILURES = 123,
+    BLOB_DB_GC_NUM_KEYS_OVERWRITTEN = 124,
+    BLOB_DB_GC_NUM_KEYS_EXPIRED = 125,
+    BLOB_DB_GC_NUM_KEYS_RELOCATED = 126,
+    BLOB_DB_GC_BYTES_OVERWRITTEN = 127,
+    BLOB_DB_GC_BYTES_EXPIRED = 128,
+    BLOB_DB_GC_BYTES_RELOCATED = 129,
+    BLOB_DB_FIFO_NUM_FILES_EVICTED = 130,
+    BLOB_DB_FIFO_NUM_KEYS_EVICTED = 131,
+    BLOB_DB_FIFO_BYTES_EVICTED = 132,
+    TXN_PREPARE_MUTEX_OVERHEAD = 133,
+    TXN_OLD_COMMIT_MAP_MUTEX_OVERHEAD = 134,
+    TXN_DUPLICATE_KEY_OVERHEAD = 135,
+    TXN_SNAPSHOT_MUTEX_OVERHEAD = 136,
+    NUMBER_MULTIGET_KEYS_FOUND = 137,
+    NO_ITERATOR_CREATED = 138,
+    NO_ITERATOR_DELETED = 139,
+    BLOCK_CACHE_COMPRESSION_DICT_MISS = 140,
+    BLOCK_CACHE_COMPRESSION_DICT_HIT = 141,
+    BLOCK_CACHE_COMPRESSION_DICT_ADD = 142,
+    BLOCK_CACHE_COMPRESSION_DICT_BYTES_INSERT = 143,
+    BLOCK_CACHE_COMPRESSION_DICT_BYTES_EVICT = 144,
```

Comment:
Titan's related metrics are missing. Before, we have added them by hand.

Reply:
They are added as a standalone enums. We may unify them using new enum or trait later.